ironfish/src/fileStores/config.ts

```diff
@@ -399,6 +418,8 @@ export class Config extends KeyStore<ConfigOptions> {
       networkId: DEFAULT_NETWORK_ID,
       customNetwork: '',
       maxSyncedAgeBlocks: 60,
+      memPoolMaxSizeBytes: 60 * MEGABYTES,
```

Comment:
Do you have any reasoning for this number? It seems large enough that it shouldn't be concerning unless we have heavy transaction volume

Reply:
Yea copying some reasoning from the doc

* 60 MB of RAM is a reasonably small amount for a personal machine to allocate for a node
* Default transaction expiration in our wallet is 15 blocks. With 30 blocks worth of transactions any default transaction that gets evicted from the mempool would not have made it into a block anyway
* 60Mb is about 60,000 single input/output transactions. When testing mempool changes in Phase 2, a node running on a M1 MacBook Pro was able to comfortably hold more than 60k transactions in the mempool (even >100) and remain synced
* Setting the limit too high is preferable to setting it too low. If it’s set too low then failure would look like transactions getting lost in the network which is a hard problem to pin down. If it is set too high then failure would be nodes running out of memory which is much easier to pin down whether it’s related to their mempool size or not